### ISSUE TITLE
Inconvenience: separate download for 10.12 and 10.13

### ISSUE THREAD

**Issue by kleuter** [crystalidea/macs-fan-control#211]
Normally the main download should be compatible with macOS 12 Sierra and later, but it's not working in 10.13: window controls are missing:
<img width="944" alt="mfc" src="https://user-images.githubusercontent.com/2600624/65493401-0f606a80-deb3-11e9-8788-330949f4c3e2.png">

Current workaround - use [legacy version](https://www.crystalidea.com/downloads/legacy/macsfancontrol.zip)

The problem is caused by the bug in the Qt Framework ([QTBUG-78731](https://bugreports.qt.io/browse/QTBUG-78731)) 
If you'd want to help - please **vote** for this issue !

We do hope that @msorvig and/or @torarnv can help get this fixed.

**Comment by kleuter** [comment]
Working on a solution

**Comment by kleuter** [comment]
The problem lies in the Qt framework we use. I submitted a bug
https://bugreports.qt.io/browse/QTBUG-78731

**Comment by StevenCee** [comment]
I'm finding the same thing, when I open it up, it's only showing one fan, when it used to show four. I tried the "legacy version workaround", but that downloaded the one I'm already using, which is not working properly, 1.5.0.3.   I tried using 1.5.0.4, but it said it doesn't work unless Mohave is installed.
I wanted to try it again, and see, but how in the world do I close the app? I don't see any place to click that will close it! If I control click the app in the Dock, it asks if I want to open it!  And there's no little dot next to it, in the Dock to even show it's already open. And then I have to go to my apps folder in the Finder, to see what version it is, as the app itself only says 1.5.0 only adding to the confusion. 

I also tried to find the guide to using it, but searched their site, and only saw bare bones information, nothing that can tell me how to really use it. I used it fine on my older computers, which also have High Sierra but it's only on this new iMac, that it's not working correctly unless there only is one fan, which is what I've been trying to find out....
